mantidimaging/core/utility/histogram.py

```diff
@@ -17,7 +17,8 @@ def set_histogram_log_scale(histogram: HistogramLUTItem) -> None:
     :param histogram: The HistogramLUTItem of an image.
     """
     x_data, y_data = histogram.plot.getData()
-    histogram.plot.setData(x_data, np.log(y_data + 1))
+    if y_data is not None:
+        histogram.plot.setData(x_data, np.log(y_data + 1))
```

Comment:
Maybe `assert isinstance(x_data, np.ndarray) and isinstance(y_data, np.ndarray)`. To be more specific and raise an error if assumption is not true.